x-pack/plugins/ingest_pipelines/public/application/components/pipeline_form/pipeline_form_fields.tsx

```diff
@@ -124,6 +131,51 @@ export const PipelineFormFields: React.FunctionComponent<Props> = ({
       >
         <PipelineEditor onLoadJson={onLoadJson} />
       </ProcessorsEditorContextProvider>
+
+      {/* _meta field */}
+      <FormRow
+        title={
+          <FormattedMessage id="xpack.ingestPipelines.form.metaTitle" defaultMessage="Metadata" />
+        }
+        description={
+          <>
+            <FormattedMessage
+              id="xpack.ingestPipelines.form.metaDescription"
+              defaultMessage="Optional metadata about the ingest pipeline. May have any contents. This map is not automatically generated by Elasticsearch."
+            />
+
+            <EuiSpacer size="m" />
+
+            <EuiSwitch
+              label={
+                <FormattedMessage
+                  id="xpack.ingestPipelines.form.metadataDescription"
```

Comment:
nit: the above formatmessage id's use `xpack.ingestPipelines.form.metaSomething` while this one uses `xpack.ingestPipelines.form.metadataSomething`. I think we should make this one more consistent with the others:

```suggestion
                  id="xpack.ingestPipelines.form.metaSwitchCaption"
```

Reply:
Done.